src/server/schema/query/index.js

```diff
@@ -1,23 +1,16 @@
 import {
   GraphQLObjectType,
-  GraphQLString
 } from 'graphql';
-import userType from '../object-types/user-type';
-import {me} from '../../db/controllers/users.controller';
+import viewerType from '../object-types/viewer-type';
+import {verifyUser} from '../../db/controllers/users.controller';
 
 const query = new GraphQLObjectType({
   name: 'Query',
   fields: {
-    hello: {
-      type: GraphQLString,
-      resolve() {
-        return 'world';
-      }
+    Viewer: {
```

Comment:
This field is a part of a Relay standard. So it should be exactly `Viewer` with the upper letter.